firmware/V3.0/Marlin/Ender 3/bltouch-for-z-homing/Configuration.h

```diff
@@ -1032,7 +1032,7 @@
  * The probe replaces the Z-MIN endstop and is used for Z homing.
  * (Automatically enables USE_PROBE_FOR_Z_HOMING.)
  */
-#define Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN
+//#define Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN
```

Comment:
@Checkers-Tigolf Why is this disabled?

Reply:
this is so the printer can use both the z endstop and the probe for homing

Reply:
Ah. That makes sense. Thanks.